### PR TITLE
Add DataSources externs

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: Cesium.externs.js [hunk 1]
```diff
@@ -2585,6 +2585,56 @@ Cesium.DataSource = function() {};
...
+ *   fill: Cesium.Color,
+ *   clampToGround: boolean
+ * }}
+ */
+Cesium.GeoJsonDataSourceLoadOptions;
+
+/**
+ * @param {string|Object} data
+ * @param {!Cesium.GeoJsonDataSourceLoadOptions} options
+ * @returns {Cesium.DataSource|Promise.<Cesium.DataSource>}
```
Comment (gberaudo) [inline Cesium.externs.js]: There should be no s in `@return`. Could you also fix the other occurences in the file?

Comment (oterral) [inline Cesium.externs.js]: Ok i can do that

**Comment by oterral** [comment]
`returns` replaced

**Comment by gberaudo** [comment]
Great, thanks.

**Pull request merged**